models/sla_policy/reply_time/int_zendesk__reply_time_business_hours.sql

```diff
@@ -17,10 +17,10 @@ with ticket_schedules as (
   select *
   from {{ ref('int_zendesk__sla_policy_applied') }}
 
-), users as (
+), reply_time as (
 
   select *
-  from {{ ref('int_zendesk__user_aggregates') }}
+  from {{ ref('int_zendesk__commenter_reply_at') }}
```

Comment:
`users` and `reply_time` consolidated in new model `int_zendesk__commenter_reply_at` since the same logic is repeated in `int_zendesk__reply_time_combined`